### DISCUSSION TITLE
Xiaomi / Mijia cameras?

### DISCUSSION THREAD

**Discussion by orangebutblue** [koush/scrypted#896] [Q&A]
I  wasn't able to find anything in regards of Xiaomi cameras in the plugins. Does that mean that my cameras aren't supported by Scrypted?

**Comment by AP0FIS** [comment]
Any news?

**Comment by NammaThalle** [comment]
yeah if anyone has any news please post it

**Comment by RexYuan** [comment]
Also interested

**Comment by davidep01** [comment]
news?

**Comment by GiorgosPap777** [comment]
Also interested

**Comment by lydigaga** [comment]
Xiaomi cameras do not support RTSP and ONVIF, so they cannot be connected

**Reply by adrianipopescu** [reply]
they use pppp, and there are already decodes to allow it to work with hass, seems like we just need a new integration to support pppp cameras

**Reply by wrysunny** [reply]
decode is where? Can parse the PPPP protocol, but the MSG_DRW message seems to use encryption/confusion. Can you provide a solution?
